docs/developer/centrifuge-sdk/guides/invest-into-a-vault/index.md

```diff
@@ -0,0 +1,104 @@
+# Invest Into a Vault
+
+This guide shows how to invest into a Centrifuge vault using the **Centrifuge SDK**.  
+It covers setup, connecting to mainnet or testnet, and sending your first investment transaction.
+
+---
+
+## Prerequisites
+
+Before you begin, make sure you have:
+
+- [Node.js](https://nodejs.org/) (v18 or later recommended)
+- A package manager: [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/)
+- A wallet or signer that can connect to Ethereum-compatible chains (e.g. MetaMask, WalletConnect, or a private key account via [Viem](https://viem.sh/))
+- Investment currency (e.g. USDC) on **mainnet** or **testnet**
+
+---
+
+## Installation
+
+Install the Centrifuge SDK in your project:
+
+```bash
+npm install @centrifuge/sdk
+```
+
+## Step 1 — Initialize the SDK
+
+Create a Centrifuge instance and connect it to mainnet:
+
+```typescript
+import Centrifuge from "@centrifuge/sdk";
+
+const centrifuge = new Centrifuge({
+  environment: "mainnet",
+});
+```
+
+> 💡 Note: For testing purposes, you can connect to testnet instead by setting environment: 'testnet'.
+
+## Step 2 — Set a Signer
+
+To send transactions, attach a signer (for example, from MetaMask or another EIP-1193 compatible provider):
+
+```typescript
+// Example: using a MetaMask injected provider
+const provider = (window as any).ethereum;
+await provider.request({ method: "eth_requestAccounts" });
+
+centrifuge.setSigner(provider);
+```
+
+If you’re using a private key or server-side setup, you can also provide a [Viem LocalAccount](https://viem.sh/docs/accounts/local).
+
+## Step 3 — Get a Pool and Vault
+
+Each pool can contain multiple share classes (vaults).
+You need the pool ID, share class ID, chain ID, and asset address.
```

Comment:
```suggestion
Each pool can contain multiple share classes and each share class can have multiple vaults issuing tokens against an deposit asset..
You need the pool ID, share class ID, chain ID, and asset Address.
```

Not sure this matches the actual API of the SDK, but I think that makes the most sense. Maybe even go 

```
const pool = await centrifuge.pool(poolId);
const token = await pool.token(tokenId)

const vault: [Addresses] = await token.vault(CHAIN_ID, ASSET_ADDRESS) // Still returns an array as we can have multiple per asset
```